cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -297,18 +298,33 @@ func (c *client) CreateMachineSetObjects(machineSets []*clusterv1.MachineSet, na
 }
 
 func (c *client) CreateMachineObjects(machines []*clusterv1.Machine, namespace string) error {
+	var (
+		wg      sync.WaitGroup
+		errOnce sync.Once
+		gerr    error
+	)
+	// the approach to concurrency here comes from golang.org/x/sync/errgroup
```

Comment:
nit: Comments should be proper sentences, starting with a capital letter and ending with a period. 

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences